burrow_exporter/metrics.go

```diff
@@ -3,9 +3,24 @@ package burrow_exporter
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
+	KafkaConsumerPartitionStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kafka_burrow_partition_status",
+			Help: "The status of the consumer on a partition as reported by burrow.",
+		},
+		[]string{"cluster", "group", "topic", "partition", "lag", "status"},
+	)
+
 	KafkaConsumerPartitionLag = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "kafka_burrow_partition_lag",
+			Help: "The lag of the consumer on a partition as reported by burrow.",
+		},
+		[]string{"cluster", "group", "topic", "partition"},
+	)
+	KafkaConsumerLastOffsetPartitionLag = prometheus.NewGaugeVec(
```

Comment:
minor: maybe this should be renamed `KafkaConsumerPartitionLastOffsetLag` to follow naming convention of the rest of metrics